docs/products/kafka/kafka-connect/howto/iceberg-sink-connector.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Create an Iceberg sink connector for Aiven for Apache Kafka®
+sidebar_label: Iceberg sink connector
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/non-swizzled/ConsoleIcons";
+
+Integrate Aiven for Apache Kafka with Apache Iceberg for real-time data ingestion into Iceberg tables.
+<!-- vale off -->
+The connector supports exactly-once delivery semantics, schema evolution, and metadata
+management, optimized for large-scale, high-performance data processing. For more
+information about the Apache Iceberg Sink Connector, see the
+[official Iceberg documentation](https://iceberg.apache.org/docs/latest/kafka-connect/#apache-iceberg-sink-connector).
+
+## Catalogs in Iceberg
+
+A catalog in Apache Iceberg serves as a layer to manage metadata related to your
+Iceberg tables. It connects the connector to storage backends, enabling
+schema management, table discovery, and configuration.
```

Comment:
This part requires some adjustments. Catalog is not connecting the connector to the storage backend, connector is writing the data to the storage directly. Catalog is used to store the metadata about the tables and for some other features. Some more info here https://iceberg.apache.org/concepts/catalog/ and we could include this as a reference for additional info.

Reply:
Thanks for this clarification! Updated this section.